static/js/context-menu.js

```diff
@@ -1208,6 +1211,10 @@ var Hover = new (class Hover {
       return this.deactivate();
     }
 
+    this.updateHoverState(elem);
+  }
+
+  updateHoverState(elem) {
```

Comment:
Can you put an underscore prefix to the method name?
This is called only internally.